app/Http/Controllers/Admin/UserController.php

```diff
@@ -86,10 +87,9 @@ public function getView(Request $request, $id)
     {
         return view('admin.users.view', [
             'user' => User::findOrFail($id),
-            'servers' => Server::select('servers.*', 'nodes.name as nodeName', 'locations.long as location')
-                ->join('nodes', 'servers.node', '=', 'nodes.id')
-                ->join('locations', 'nodes.location', '=', 'locations.id')
-                ->where('owner', $id)
+            'servers' => Server::select('servers.*')
+                ->whereIn('servers.id', Subuser::select('server_id')->where('user_id', $id)->get())
```

Comment:
So I think this is going to break as soon as we implement #286, but as soon as that is implemented there should be more straightforward logic for this.